Fix Issue With Multiple Values in src / dst of DFW Rule

mattallford opened the pull request:
Implement fix to capture and commar separate multiple values in the source, destination, source type and destination type fields

ryan-jan commented:
Instead of `## [0.1.2] - 2018-09-22` I think we are going to keep a running `## [Unreleased]` section at the top of the `CHANGELOG` and then when the version number gets increased we will update it and start a new unreleased section. Could you tweak that in this PR?

mattallford committed:
Fix Issue With Multiple Values in src / dst of DFW Rule

Implement fix to capture and comma separate multiple values in the source, destination, source type and destination type fields
Fix typo

mattallford commented:
All sorted @ryan-jan !
